pkg/services/resource.go

```diff
@@ -199,8 +200,11 @@ func (s *sqlResourceService) UpdateStatus(ctx context.Context, resource *api.Res
 		return nil, false, errors.GeneralError("Unable to convert resource status to cloudevent: %s", err)
 	}
 
-	logger.Info("Updating resource status")
-	logger.V(4).Info("Updating resource status", "event", resourceStatusEvent)
+	logger.Info("Updating resource status", "context", util.FormatEventContext(resourceStatusEvent.Context))
```

Comment:
we may use the same format as the sdk-go https://github.com/open-cluster-management-io/sdk-go/blob/main/pkg/cloudevents/generic/clients/baseclient.go#L147

and the source I think we need to add here

Reply:
besides event extensions, do we also need log event type, source and id?

Reply:
code updated.